docs/AUTHORS.md

```diff
@@ -17,5 +17,6 @@
 <!-- Please write your name alphabetically and use the below template. -->
 <!-- - First Last ([@username](https://github.com/username)) <example@email.com> -->
 
+- Pierre Mourlanne ([@pmourlanne](https://github.com/pmourlanne)) <pmourlanne@gmail.com>
```

Comment:
I put my name first because of alphabetical order. If you want to put it below the earlier contributor that's fine by me, you tell me :)

Reply:
You did it right!